docs/Server-configuration.md

```diff
@@ -93,3 +93,66 @@ IMPOSTOR_Server__ListenPort=22023
 IMPOSTOR_AntiCheat__Enabled=true
 IMPOSTOR_AntiCheat__BanIpFromGame=true
 ```
+
+# systemd
+
+Linux users will likely want to create a systemd service definition to manage running the Impostor application.  Benefits to using a systemd service include the following:
+
+- The server process can automatically restart if the process dies
+- Impostor can be set to start on boot (handy if you do system updates regularly)
+- You can use systemd targets to ensure the process doesn't start until the system dependencies are met
+
+Using your favorite text editor, create the following service file:
+
+```
+sudo vim /etc/systemd/system/impostor.service
+```
+
+Modify the following template to your needs and populate the service file.
```

Comment:
Should we put the template in here or should we put it as a separate file that people can just copypaste? (or download from github and use directly)